apps/prairielearn/python/question_phases.py

```diff
@@ -173,8 +162,43 @@ def process_element_return_none(element: lxml.html.HtmlElement) -> None:
     if phase == "file":
         result = filelike_to_string(result)
 
-    # We may have added an `extensions` property to the `data` object; remove it.
-    if "extensions" in data:
-        del data["extensions"]
-
     return result, processed_elements
+
+
+def prepare_data(
```

Comment:
This new function makes the code look much cleaner!

Reply:
Agreed!